src/components/FooterNavList/FooterNavList.tsx

```diff
@@ -0,0 +1,31 @@
+import { ReactElement } from 'react';
+import Link from 'next/link';
+
+interface FooterNavListProps {
+  navTitle: string;
+  navList: Array<{ title: string; href: string }>;
```

Comment:
I think you need a separate type like LinkOptions, place it in the `src/types/link.ts` file. You can reuse it in the Footer component as well

Reply:
@ain-odin don't forget about this one🙂

Reply:
<img width="662" alt="Screenshot 2023-09-02 at 12 13 34" src="https://github.com/Alex-2kZharkov/oldboy-barbershop/assets/119830208/bc1ec435-30a8-417e-8eaf-ec9d1892e9de">